src/Concordium/Client/Runner.hs

```diff
@@ -3767,7 +3767,12 @@ processLegacyCmd action backend =
       withClient backend $
         readBlockHashOrDefault Best block >>=
           getCryptographicParameters >>=
-            printResponseValueAsJSON
+            printResponseValueAsJSON . fmap (fmap (\respVal ->
+             object [
+                "v" AE..= (0 :: Integer),
+                "value" AE..= respVal
+              ]
+            ))
```

Comment:
```suggestion
            printResponseValueAsJSON . fmap (fmap (Versioned VERSION_0))
```

Please don't do manual JSON construction. Use the Versioned struct

Reply:
Neat. I was unaware of this.